docs/INSTALL-devel-flowchart.txt

```diff
@@ -0,0 +1,57 @@
+
+
+	---------------------
+	| Install Packages  |
+	| Required for HaaS |
+	---------------------
+		  #
+		  #
+		  #
+		  #		     
+                #   #   	    				################
+	      #	      #     NO     			       # Install	#
+	    # 	SQLite  # # # # # # # # # # # # # # # # # # # #  PostgreSQL DB   #		    
+	      #   DB  #		   			       #		#	
+	        #   #		    				 ###############
+                  #							#
+		  #  YES						#
+		  #							#
+		  #							#
+	--------------------------			--------------------------------------
+	|  Set up python virtual |			|  Create a systems user: "haas_dev" |
+	|  Environment	         |<-----|		--------------------------------------
+ 	--------------------------	|				#
+		  #		 	|				#
+		  #			|				#
+		  #			|				#
+	-------------------		|		---------------------------------
+	| GIT clone HaaS |		|		| Create a DB role: "haas_dev"  |
+	-------------------		|		| Create a database: "haas"     |
+		 #			|		---------------------------------
+		 #			|				#
+		 #		 	|				#
+	---------------------		|		-------------------
+	| Configur haas.cfg |		----------------|  cd ~/haas_dev  |
+	---------------------				-------------------
+		 #
+		 #	
+		 #
+	---------------------------
+	| Initialize the database |
+	---------------------------
+		 #
+		 #
+		 #	
+	---------------------	
+	| Start HaaS Server |
```

Comment:
Should that be "Servers", since "haas serve_networks" in addition to "haas serve"?